CMakeLists.txt

```diff
@@ -560,6 +562,11 @@ foreach(_scheduler ${HPX_WITH_THREAD_SCHEDULERS_UC})
     hpx_add_config_define(HPX_HAVE_PERIODIC_PRIORITY_SCHEDULER)
     set(HPX_WITH_PERIODIC_PRIORITY_SCHEDULER ON CACHE INTERNAL "")
   endif()
+  # The throttling scheduler has not been tested neither on Windows nor on Mac
```

Comment:
Please enable this option only for the platforms you tested on. If it's untested (or unsupported) on Windows, Mac, etc. we should make sure the option produces a configuration error.